src/MSALWrapper.Test/AuthFlow/BrokerTest.cs

```diff
@@ -172,27 +176,26 @@ public async Task BrokerAuthFlow_GetTokenSilent_MsalServiceException()
 
             // Assert - this method should not throw for known types of excpeptions, instead return null, so
             // our caller can retry auth another way.
-            this.pcaWrapperMock.VerifyAll();
             authFlowResult.TokenResult.Should().Be(null);
             authFlowResult.Errors.Should().HaveCount(1);
             authFlowResult.Errors[0].Should().BeOfType(typeof(MsalServiceException));
             authFlowResult.AuthFlowName.Should().Be("broker");
         }
 
         [Test]
-        public async Task BrokerAuthFlow_GetTokenSilent_OperationCanceledException()
+        public async Task BrokerAuthFlow_GetTokenSilent_OperationCanceledException_ThenSucceeds()

```

Comment:
Should we have a corresponding `ThenFails` test? Or maybe a `ThenMSALReturnsNull`?

Reply:
The primary difference is already tested where silent auth doesn't work - and interactive auth also doesn't work, returning null or throwing exceptions - both of those are covered in tests below.